### ISSUE TITLE
When my access token is being created for one model, all other Api guards become athenticated

### ISSUE THREAD

**Issue by mobrad1** [sfelix-martins/passport-multiauth#51]
When i run auth->check for each guard
```json
  "hospital-api" => true
  "doctor-api" => true
  "pharmacy-api" => true
  "admin-api" => false
  "patient-api" => true

```

**Comment by sfelix-martins** [comment]
What’s version are you using?

**Comment by mobrad1** [comment]
I'm making use of laravel 5.7

**Comment by sfelix-martins** [comment]
Good, and the version from laravel passport and this package ?

**Comment by sfelix-martins** [comment]
@mobrad1 
Did you identify the problem? 

```
if yes {
     please share the solution so that it helps other people. 
} else {
    I need more details of your implementation to solve the problem.
}
```

Thanks!

**Comment by sfelix-martins** [comment]
This issue should be solved by the merge #71
